users/klackygears/ReadMe.md

```diff
@@ -0,0 +1,15 @@
+Copyright<2019><James Smith><bronzegears@gmail.com> @klackygears
```

Comment:
Could you rename this file to lowercase?

Reply:
woops! yep, got it!